src/main/java/Duke.java

```diff
@@ -1,10 +1,73 @@
+import java.io.IOException;
+import duke.exceptions.DukeException;
+import duke.managers.Parser;
+import duke.managers.Storage;
+import duke.managers.TaskList;
+import duke.managers.Ui;
+import duke.commands.Command;
+
 public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Initialises a new Duke object including new objects such as storage, ui and task list.
```

Comment:
Need a new line separating description of method/class and the @param @return tags for JavaDocs